cmd/commands.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/ardikabs/shorty/kutt"
+	"github.com/spf13/cobra"
+)
+
+var api kutt.API
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+	rootCmd.AddCommand(submitCmd)
+	rootCmd.AddCommand(deleteCmd)
+
+	apiToken := os.Getenv("KUTT_TOKEN")
+	if apiToken == "" {
+		fmt.Println("Kutt API Token are not set. 'KUTT_TOKEN'")
```

Comment:
this means rejecting any user input if `KUTT_TOKEN` is N/A; 

Should reject on respective function instead of on invoke